RecoPixelVertexing/PixelTriplets/python/quadrupletseedmerging_cff.py

```diff
@@ -62,4 +62,4 @@
 from Configuration.Eras.Modifier_trackingPhase1PU70_cff import trackingPhase1PU70
 trackingPhase1PU70.toModify(PixelSeedMergerQuadruplets, **_forPhase1)
 from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
-trackingPhase2PU140.toModify(PixelSeedMergerQuadruplets, layerList = _layerListForPhase2)
+trackingPhase2PU140.toModify(PixelSeedMergerQuadruplets, layerList = _layerListForPhase2, **_forPhase1)
```

Comment:
In practice this keeps the framework circular dependency checker happy when running on scheduled mode (that we discussed in an e-mail thread few weeks ago). See also the comment above on lines 51-55.